testutils/lint: TestNightlyLint failed

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15194486?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15194486?buildTab=artifacts#/) on master @ [16d41751607b92234351c1ab27053c3875a4f2b7](https://github.com/cockroachdb/cockroach/commits/16d41751607b92234351c1ab27053c3875a4f2b7):


```
             183:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1464:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             1674:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             1558:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1726:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1716:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             2194:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             641:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             2096:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             890:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1498:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             518:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             1052:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1298:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1385:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             2119:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             1093:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             941:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             25:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1762:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             458:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             2011:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             858:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             2109:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1237:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             2207:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             136:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.60s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-38603

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15213443?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15213443?buildTab=artifacts#/) on master @ [59b261a579cbe2c032a5dd3e182ff67aeee900b9](https://github.com/cockroachdb/cockroach/commits/59b261a579cbe2c032a5dd3e182ff67aeee900b9):


```
             1058:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             303:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1757:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             1507:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1360:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             462:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             363:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             843:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1628:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             2217:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             333:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1255:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1142:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             580:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             12:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             980:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1208:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             2183:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             1548:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             214:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             39:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             2034:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1312:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             243:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1322:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             1120:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1902:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.53s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15224243?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15224243?buildTab=artifacts#/) on master @ [4c2e7761acd050aaee565443932b6b0eca55620b](https://github.com/cockroachdb/cockroach/commits/4c2e7761acd050aaee565443932b6b0eca55620b):


```
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1981:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1384:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             726:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             2187:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             938:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             344:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             836:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             376:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1405:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             495:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1908:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             537:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1108:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             1757:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1243:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             78:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             667:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             764:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             2144:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             1343:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             19:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1536:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             324:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             2203:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1083:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.60s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15235279?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15235279?buildTab=artifacts#/) on master @ [4cc0bfcc14771331fea57de01e1ea78b07393f3d](https://github.com/cockroachdb/cockroach/commits/4cc0bfcc14771331fea57de01e1ea78b07393f3d):


```
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             1439:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             921:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             1048:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1538:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             268:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             911:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1658:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             97:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             699:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             893:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1226:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1258:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             60:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1727:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1737:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             564:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             1168:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             525:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             939:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1029:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             358:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             2148:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             483:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             2162:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             860:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1501:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             278:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15254572?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15254572?buildTab=artifacts#/) on master @ [b185402d34768c898d63a61de3e585e4723fbe81](https://github.com/cockroachdb/cockroach/commits/b185402d34768c898d63a61de3e585e4723fbe81):


```
             1560:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             2085:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1821:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1096:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1352:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             432:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             451:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             974:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1156:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             107:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             2156:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             1746:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             240:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             260:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             994:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             793:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             984:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             829:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1139:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             2109:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             696:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             633:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             675:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1275:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             472:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             2004:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1927:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15273552?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15273552?buildTab=artifacts#/) on master @ [6300c3c3367ad46ac48bf24915cf0d73cae446a0](https://github.com/cockroachdb/cockroach/commits/6300c3c3367ad46ac48bf24915cf0d73cae446a0):


```
             195:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             779:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             924:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1074:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1290:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             555:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             2040:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             734:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1684:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1295:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             2100:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1422:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             2192:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             414:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             1914:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1871:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             94:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             751:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             318:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             633:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1007:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1362:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             825:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             1806:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1635:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             1606:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1578:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.83s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15288605?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15288605?buildTab=artifacts#/) on master @ [992d4573fb8b71e28717b4d14c4c7ebccb38c412](https://github.com/cockroachdb/cockroach/commits/992d4573fb8b71e28717b4d14c4c7ebccb38c412):


```
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             177:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             134:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             729:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1598:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             1770:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             1413:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1614:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             999:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1942:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             2181:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1356:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             966:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1244:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1479:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1780:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             2230:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             1926:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             9:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1837:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             216:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             2114:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1093:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             2218:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             422:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1827:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             1624:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15301923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15301923?buildTab=artifacts#/) on master @ [855b9cc97afa3df4f7e17f928c04ab0834b2630c](https://github.com/cockroachdb/cockroach/commits/855b9cc97afa3df4f7e17f928c04ab0834b2630c):


```
             2189:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             2207:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             233:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             481:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             676:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1237:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             104:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             511:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             1597:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1854:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             43:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             310:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             750:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             938:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1058:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1643:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             2159:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1647:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             1503:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             200:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             634:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             646:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             2110:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             1176:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             447:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             659:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             941:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1679:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             1512:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             1111:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15314197?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15314197?buildTab=artifacts#/) on master @ [93ad913106b6f0f6ec98bc2cfa788ff6d8085bd4](https://github.com/cockroachdb/cockroach/commits/93ad913106b6f0f6ec98bc2cfa788ff6d8085bd4):


```
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             876:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1674:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             2091:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             1879:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             458:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             337:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1002:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             1218:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             1955:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             2139:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1102:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             2203:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             294:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             906:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             268:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             310:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             834:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             168:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             577:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             655:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             778:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             987:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1208:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1829:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1059:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             2237:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1832:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             794:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15320946?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15320946?buildTab=artifacts#/) on master @ [93ad913106b6f0f6ec98bc2cfa788ff6d8085bd4](https://github.com/cockroachdb/cockroach/commits/93ad913106b6f0f6ec98bc2cfa788ff6d8085bd4):


```
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             1854:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             246:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             795:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             562:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             2163:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1155:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             2013:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             180:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             584:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             781:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             109:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             1327:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             944:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             931:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1167:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1922:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             17:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1615:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             376:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1515:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             969:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             441:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             1265:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             542:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (2.65s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15327662?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15327662?buildTab=artifacts#/) on master @ [5d013285fa696f53df2abb39f44ffc777125fe1c](https://github.com/cockroachdb/cockroach/commits/5d013285fa696f53df2abb39f44ffc777125fe1c):


```
             245:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1178:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1370:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1452:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1558:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1755:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             411:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             2008:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             996:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1853:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1645:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             466:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             1042:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             324:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             362:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             433:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1455:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1759:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1188:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1262:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1962:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             141:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             2047:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1099:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1147:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             618:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             1197:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.60s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15340208?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15340208?buildTab=artifacts#/) on master @ [d146ecff6f687e438706cf63591cafca60cc116d](https://github.com/cockroachdb/cockroach/commits/d146ecff6f687e438706cf63591cafca60cc116d):


```
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             2003:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1317:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1223:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             337:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             721:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             1637:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1695:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             1127:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             796:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             2087:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             671:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1278:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             1627:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             1537:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             25:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1932:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             186:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             762:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             986:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1440:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1566:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1999:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             2232:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             569:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1601:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1297:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1012:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             927:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15356586?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15356586?buildTab=artifacts#/) on master @ [8156d095961d7271df61fe2e2f716ef072bc5f96](https://github.com/cockroachdb/cockroach/commits/8156d095961d7271df61fe2e2f716ef072bc5f96):


```
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             1739:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             2163:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1519:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             324:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             2038:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1184:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             349:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             2048:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             556:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             799:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             1856:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             1922:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             764:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             692:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1637:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             158:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1751:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             1777:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1425:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             486:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             1053:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             1563:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1140:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1086:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.67s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15371552?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15371552?buildTab=artifacts#/) on master @ [c580e634736b2d2b6da544eecf16664d3caca740](https://github.com/cockroachdb/cockroach/commits/c580e634736b2d2b6da544eecf16664d3caca740):


```
             656:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1169:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             1439:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             2237:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             2073:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1499:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             320:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             1239:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             908:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             1989:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             578:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             686:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1938:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             394:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             1999:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             2143:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             504:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             187:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1559:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             1717:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             830:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             177:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             290:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             876:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1546:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             29:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             1365:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1961:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.68s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15386478?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15386478?buildTab=artifacts#/) on master @ [c7f16c96d4db6ef97085f972f094c7e4e7c1518f](https://github.com/cockroachdb/cockroach/commits/c7f16c96d4db6ef97085f972f094c7e4e7c1518f):


```
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             227:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             1620:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             237:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1298:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             144:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             807:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             2072:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             1021:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1815:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1165:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             201:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             1588:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             482:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             86:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             951:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             1322:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1598:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             45:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             459:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             2081:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             1906:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             75:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             402:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             25:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             706:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (5.25s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15399678?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15399678?buildTab=artifacts#/) on master @ [b5081e997adaf5950c9b85dfb6655f226bf16d29](https://github.com/cockroachdb/cockroach/commits/b5081e997adaf5950c9b85dfb6655f226bf16d29):


```
             435:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             913:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             27:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             510:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             482:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             567:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             334:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             2217:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             606:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             445:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             207:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1021:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             262:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             1693:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             1348:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             740:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             9:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             722:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             1842:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1501:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1490:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             170:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             2167:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             293:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             2190:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             2065:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.72s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15417791?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15417791?buildTab=artifacts#/) on master @ [0ea8b622c5354a891cfb867d8d872a0746ba77b3](https://github.com/cockroachdb/cockroach/commits/0ea8b622c5354a891cfb867d8d872a0746ba77b3):


```
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             357:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1814:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1764:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             1382:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             507:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             842:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             101:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             1063:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1054:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1360:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1372:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             1642:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1186:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1522:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             1574:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             990:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             934:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             952:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1625:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             2139:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             1422:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1212:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             2113:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1148:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15427547?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15427547?buildTab=artifacts#/) on master @ [0ea8b622c5354a891cfb867d8d872a0746ba77b3](https://github.com/cockroachdb/cockroach/commits/0ea8b622c5354a891cfb867d8d872a0746ba77b3):


```
             582:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             1262:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             2030:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1177:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             1520:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             1228:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             565:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1619:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             702:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             728:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             2070:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1809:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             220:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             826:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             245:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             2167:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             1122:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1831:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             299:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             895:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1113:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1296:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             349:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             1271:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             363:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             312:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1187:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.57s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15444707?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15444707?buildTab=artifacts#/) on master @ [d5f825ce04362a75781b75e19cf429f2ed5ed61b](https://github.com/cockroachdb/cockroach/commits/d5f825ce04362a75781b75e19cf429f2ed5ed61b):


```
             2211:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             2201:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1231:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             447:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             2158:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             776:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             1589:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             33:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1454:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             24:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             719:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             458:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1129:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             995:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             269:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1796:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1639:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             2051:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             1972:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             1599:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             1088:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             797:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             1715:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             1015:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             1467:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             2103:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15454927?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15454927?buildTab=artifacts#/) on master @ [e5f5597b1c1570fff03ee6dfd4a1e8be65c0e008](https://github.com/cockroachdb/cockroach/commits/e5f5597b1c1570fff03ee6dfd4a1e8be65c0e008):


```
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             1168:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             1049:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             1347:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             1241:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1411:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             584:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             2065:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             849:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             640:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             781:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             2201:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             952:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             217:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             1177:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             186:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             550:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             350:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1322:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             1956:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             140:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1655:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             149:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             73:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1332:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             2159:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.73s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15465778?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15465778?buildTab=artifacts#/) on master @ [64bd4bbac8f8d27a2fe4968404cc3d048480375f](https://github.com/cockroachdb/cockroach/commits/64bd4bbac8f8d27a2fe4968404cc3d048480375f):


```
             987:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1671:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1696:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             838:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             997:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             1463:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1180:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1039:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             52:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             261:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             61:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1654:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             580:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             929:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             1598:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1405:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             2237:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1995:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             1917:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1303:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1404:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             1927:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             2016:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1883:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             1573:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1846:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.74s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15478992?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15478992?buildTab=artifacts#/) on master @ [d77b6e709e7d90625ff31bc0016eed8c7a6dc65a](https://github.com/cockroachdb/cockroach/commits/d77b6e709e7d90625ff31bc0016eed8c7a6dc65a):


```
             707:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1308:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1799:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1811:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             2237:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             1649:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             866:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1273:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             970:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             1554:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1066:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             267:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             377:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             547:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             593:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1648:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1857:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             2185:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1395:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             1490:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             2222:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             552:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1028:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             157:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             402:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             1219:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1500:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             1964:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             1262:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.62s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15484127?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15484127?buildTab=artifacts#/) on master @ [a21a5fc2c955415a5e1b44cf0d995e1048f667c9](https://github.com/cockroachdb/cockroach/commits/a21a5fc2c955415a5e1b44cf0d995e1048f667c9):


```
             2236:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1353:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             607:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             1893:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             813:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             2145:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             2019:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             40:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             1270:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             1341:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             272:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1593:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             543:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             258:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             643:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1632:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             1728:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             664:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1499:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             504:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             839:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             377:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             990:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             21:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1729:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.65s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15491072?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15491072?buildTab=artifacts#/) on master @ [c84a89569fab4cc4896a0234bec1e1822dd420d8](https://github.com/cockroachdb/cockroach/commits/c84a89569fab4cc4896a0234bec1e1822dd420d8):


```
             683:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1534:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             831:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             1941:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1680:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             1868:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             1271:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             274:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1154:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1483:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             232:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             415:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             522:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             840:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             2064:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             2183:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             248:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1998:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1294:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             2124:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1742:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             1518:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             424:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             2213:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             364:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             2051:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.53s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15504718?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15504718?buildTab=artifacts#/) on master @ [3d9eaa7939e4fb5aa0518ba7e19750a55c93cf11](https://github.com/cockroachdb/cockroach/commits/3d9eaa7939e4fb5aa0518ba7e19750a55c93cf11):


```
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             1736:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1052:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             544:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             222:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             237:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             518:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             828:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1048:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1257:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1664:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             2066:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1380:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             2091:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             2177:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1669:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1473:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             488:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             977:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             497:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             1490:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             931:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             2237:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1126:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             103:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1916:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             958:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             1446:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             701:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.64s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15519704?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15519704?buildTab=artifacts#/) on master @ [4c06ddd826bd32f84adbdf2f78f90c7d0c2f2d60](https://github.com/cockroachdb/cockroach/commits/4c06ddd826bd32f84adbdf2f78f90c7d0c2f2d60):


```
             2099:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1936:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             55:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/create-procedure.html : 404 Not Found
             311:  https://www.cockroachlabs.com/docs/v24.2/create-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1913:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             1350:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             738:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             2040:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             37:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1687:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-table.html : 404 Not Found
             410:  https://www.cockroachlabs.com/docs/v24.2/alter-table.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             423:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1228:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             277:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1722:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             209:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             146:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             46:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1218:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             687:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1369:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-ranges.html : 404 Not Found
             1334:  https://www.cockroachlabs.com/docs/v24.2/show-ranges.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1526:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             2142:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             474:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.66s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15535946?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15535946?buildTab=artifacts#/) on master @ [d0582983521533ffc15a8b8525a188d80da2a0b7](https://github.com/cockroachdb/cockroach/commits/d0582983521533ffc15a8b8525a188d80da2a0b7):


```
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             789:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/drop-database.html : 404 Not Found
             249:  https://www.cockroachlabs.com/docs/v24.2/drop-database.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             2041:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             86:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1010:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1161:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1261:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             228:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             223:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             240:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             589:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             833:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1336:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1413:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1826:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1340:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/show-grants.html : 404 Not Found
             1559:  https://www.cockroachlabs.com/docs/v24.2/show-grants.html
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             1982:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             2155:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             813:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             135:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             227:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/explain.html : 404 Not Found
             680:  https://www.cockroachlabs.com/docs/v24.2/explain.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1243:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1416:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             2165:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             271:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html : 404 Not Found
             1426:  https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
             1584:https://www.cockroachlabs.com/docs/v24.2/cluster-settings.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.59s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15551309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15551309?buildTab=artifacts#/) on master @ [802554ac59763c31aee034651efcd28f7bdcdd5d](https://github.com/cockroachdb/cockroach/commits/802554ac59763c31aee034651efcd28f7bdcdd5d):


```
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             1656:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html : 404 Not Found
             1940:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/set-transaction.html : 404 Not Found
             1198:https://www.cockroachlabs.com/docs/v24.2/set-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             53:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             938:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1391:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1300:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/create-index.html : 404 Not Found
             352:https://www.cockroachlabs.com/docs/v24.2/create-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             1086:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/create-table-as.html : 404 Not Found
             1530:https://www.cockroachlabs.com/docs/v24.2/create-table-as.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1888:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             852:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             953:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/alter-function.html : 404 Not Found
             90:  https://www.cockroachlabs.com/docs/v24.2/alter-function.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             159:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             2215:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             785:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1136:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1239:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1253:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1423:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1525:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1926:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             1073:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             14:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             375:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/drop-function.html : 404 Not Found
             805:  https://www.cockroachlabs.com/docs/v24.2/drop-function.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             1608:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (4.58s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15565200?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15565200?buildTab=artifacts#/) on master @ [c7f37e813dbd1e6d2afdb88c5e9cd986dc061299](https://github.com/cockroachdb/cockroach/commits/c7f37e813dbd1e6d2afdb88c5e9cd986dc061299):


```
             2120:  https://www.cockroachlabs.com/docs/v24.2/alter-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/show-databases.html : 404 Not Found
             2066:  https://www.cockroachlabs.com/docs/v24.2/show-databases.html
        https://www.cockroachlabs.com/docs/v24.2/alter-view.html : 404 Not Found
             2106:  https://www.cockroachlabs.com/docs/v24.2/alter-view.html
        https://www.cockroachlabs.com/docs/v24.2/show-tables.html : 404 Not Found
             1516:  https://www.cockroachlabs.com/docs/v24.2/show-tables.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             1114:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/show-columns.html : 404 Not Found
             796:  https://www.cockroachlabs.com/docs/v24.2/show-columns.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             999:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/select-clause.html : 404 Not Found
             387:  https://www.cockroachlabs.com/docs/v24.2/select-clause.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             2004:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             1453:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1643:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1340:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/savepoint.html : 404 Not Found
             1228:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
             1877:  https://www.cockroachlabs.com/docs/v24.2/savepoint.html
        https://www.cockroachlabs.com/docs/v24.2/show-vars.html : 404 Not Found
             507:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
             1218:  https://www.cockroachlabs.com/docs/v24.2/show-vars.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1867:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             722:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1544:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             641:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1507:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             1790:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             134:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/truncate.html : 404 Not Found
             1963:  https://www.cockroachlabs.com/docs/v24.2/truncate.html
        https://www.cockroachlabs.com/docs/v24.2/revoke.html : 404 Not Found
             1406:  https://www.cockroachlabs.com/docs/v24.2/revoke.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             609:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.55s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15574160?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15574160?buildTab=artifacts#/) on master @ [c7f37e813dbd1e6d2afdb88c5e9cd986dc061299](https://github.com/cockroachdb/cockroach/commits/c7f37e813dbd1e6d2afdb88c5e9cd986dc061299):


```
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1246:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             2214:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        https://www.cockroachlabs.com/docs/v24.2/create-function.html : 404 Not Found
             1908:  https://www.cockroachlabs.com/docs/v24.2/create-function.html
        https://www.cockroachlabs.com/docs/v24.2/alter-database.html : 404 Not Found
             1669:  https://www.cockroachlabs.com/docs/v24.2/alter-database.html
        https://www.cockroachlabs.com/docs/v24.2/alter-index.html : 404 Not Found
             157:  https://www.cockroachlabs.com/docs/v24.2/alter-index.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             1427:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html : 404 Not Found
             1774:  https://www.cockroachlabs.com/docs/v24.2/show-zone-configurations.html
        https://www.cockroachlabs.com/docs/v24.2/show-users.html : 404 Not Found
             1257:  https://www.cockroachlabs.com/docs/v24.2/show-users.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             903:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/drop-index.html : 404 Not Found
             400:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
             1131:  https://www.cockroachlabs.com/docs/v24.2/drop-index.html
        https://www.cockroachlabs.com/docs/v24.2/set-vars.html : 404 Not Found
             126:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             587:https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             603:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
             1152:  https://www.cockroachlabs.com/docs/v24.2/set-vars.html
        https://www.cockroachlabs.com/docs/v24.2/alter-partition.html : 404 Not Found
             89:  https://www.cockroachlabs.com/docs/v24.2/alter-partition.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             1236:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/import-into.html : 404 Not Found
             929:  https://www.cockroachlabs.com/docs/v24.2/import-into.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             178:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html : 404 Not Found
             1528:  https://www.cockroachlabs.com/docs/v24.2/rollback-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1649:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/grant.html : 404 Not Found
             978:  https://www.cockroachlabs.com/docs/v24.2/grant.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1763:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             485:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             1087:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             1487:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.46s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

testutils/lint.TestNightlyLint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15583832?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/15583832?buildTab=artifacts#/) on master @ [bd4c5fd7c65c0a9afa8ba05675e49747a4f68881](https://github.com/cockroachdb/cockroach/commits/bd4c5fd7c65c0a9afa8ba05675e49747a4f68881):


```
        https://www.cockroachlabs.com/docs/v24.2/delete.html : 404 Not Found
             1490:  https://www.cockroachlabs.com/docs/v24.2/delete.html
        https://www.cockroachlabs.com/docs/v24.2/show-index.html : 404 Not Found
             1928:  https://www.cockroachlabs.com/docs/v24.2/show-index.html
        https://www.cockroachlabs.com/docs/v24.2/create-view.html : 404 Not Found
             1194:  https://www.cockroachlabs.com/docs/v24.2/create-view.html
        https://www.cockroachlabs.com/docs/v24.2/upsert.html : 404 Not Found
             1663:  https://www.cockroachlabs.com/docs/v24.2/upsert.html
        https://www.cockroachlabs.com/docs/v24.2/show-constraints.html : 404 Not Found
             1088:  https://www.cockroachlabs.com/docs/v24.2/show-constraints.html
        https://www.cockroachlabs.com/docs/v24.2/update.html : 404 Not Found
             664:  https://www.cockroachlabs.com/docs/v24.2/update.html
        https://www.cockroachlabs.com/docs/v24.2/alter-type.html : 404 Not Found
             23:  https://www.cockroachlabs.com/docs/v24.2/alter-type.html
        https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html : 404 Not Found
             1842:  https://www.cockroachlabs.com/docs/v24.2/commit-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/show-create.html : 404 Not Found
             889:  https://www.cockroachlabs.com/docs/v24.2/show-create.html
        https://www.cockroachlabs.com/docs/v24.2/create-database.html : 404 Not Found
             1311:  https://www.cockroachlabs.com/docs/v24.2/create-database.html
        https://www.cockroachlabs.com/docs/v24.2/show-backup.html : 404 Not Found
             169:  https://www.cockroachlabs.com/docs/v24.2/show-backup.html
        https://www.cockroachlabs.com/docs/v24.2/show-partitions.html : 404 Not Found
             1428:  https://www.cockroachlabs.com/docs/v24.2/show-partitions.html
        https://www.cockroachlabs.com/docs/v24.2/insert.html : 404 Not Found
             1564:  https://www.cockroachlabs.com/docs/v24.2/insert.html
        https://www.cockroachlabs.com/docs/v24.2/table-expressions.html : 404 Not Found
             127:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             214:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             558:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             660:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1042:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             1191:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
             2236:  https://www.cockroachlabs.com/docs/v24.2/table-expressions.html
        https://www.cockroachlabs.com/docs/v24.2/create-table.html : 404 Not Found
             562:https://www.cockroachlabs.com/docs/v24.2/create-table.html
        https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html : 404 Not Found
             1607:  https://www.cockroachlabs.com/docs/v24.2/begin-transaction.html
        https://www.cockroachlabs.com/docs/v24.2/restore.html : 404 Not Found
             1248:  https://www.cockroachlabs.com/docs/v24.2/restore.html
        https://www.cockroachlabs.com/docs/v24.2/show-default-privileges : 404 Not Found
             921:  https://www.cockroachlabs.com/docs/v24.2/show-default-privileges
        https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html : 404 Not Found
             380:  https://www.cockroachlabs.com/docs/v24.2/drop-procedure.html
        https://www.cockroachlabs.com/docs/v24.2/backup.html : 404 Not Found
             711:  https://www.cockroachlabs.com/docs/v24.2/backup.html
        https://www.cockroachlabs.com/docs/v24.2/drop-table.html : 404 Not Found
             1361:  https://www.cockroachlabs.com/docs/v24.2/drop-table.html
        54 errors
    --- FAIL: TestNightlyLint/TestHelpURLs (0.54s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestNightlyLint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/build/info.go
+++ b/pkg/build/info.go
@@ -107,13 +107,17 @@ func BinaryVersion() string {
 	return binaryVersion
 }
 
-// BinaryVersionPrefix returns the version prefix of the current build.
-func BinaryVersionPrefix() string {
-	v, err := version.Parse(BinaryVersion())
-	if err != nil {
-		return "dev"
+// VersionForURLs is used to determine the version to use in public-facing doc URLs.
+// It returns "vX.Y" for all release versions, and all prerelease versions >= "alpha.1".
+// X and Y are the major and minor, respectively, of the version specified in version.txt.
+// For all other prerelease versions, it returns "dev".
+// N.B. new public-facing doc URLs are expected to be up beginning with the "alpha.1" prerelease. Otherwise, "dev" will
+// cause the url mapper to redirect to the latest stable release.
+func VersionForURLs() string {
+	if parsedVersionTxt.PreRelease() >= "alpha.1" {
+		return fmt.Sprintf("v%d.%d", parsedVersionTxt.Major(), parsedVersionTxt.Minor())
 	}
-	return fmt.Sprintf("v%d.%d", v.Major(), v.Minor())
+	return "dev"
 }
 
 // BranchReleaseSeries returns tha major and minor in version.txt, without
@@ -211,5 +215,5 @@ func TestingOverrideVersion(v string) func() {
 
 // MakeIssueURL produces a URL to a CockroachDB issue.
 func MakeIssueURL(issue int) string {
-	return fmt.Sprintf("https://go.crdb.dev/issue-v/%d/%s", issue, BinaryVersionPrefix())
+	return fmt.Sprintf("https://go.crdb.dev/issue-v/%d/%s", issue, VersionForURLs())
 }

--- a/pkg/clusterversion/cockroach_versions_test.go
+++ b/pkg/clusterversion/cockroach_versions_test.go
@@ -169,7 +169,8 @@ func TestReleaseSeries(t *testing.T) {
 	}
 
 	// Verify the latest version.
-	require.Equal(t, fmt.Sprintf("v%s", Latest.ReleaseSeries()), build.BinaryVersionPrefix())
+	major, minor := build.BranchReleaseSeries()
+	require.Equal(t, fmt.Sprintf("v%s", Latest.ReleaseSeries()), fmt.Sprintf("v%d.%d", major, minor))
 
 	// Verify the ReleaseSeries results down to MinSupported.
 	expected := Latest.ReleaseSeries()

--- a/pkg/docs/docs.go
+++ b/pkg/docs/docs.go
@@ -18,12 +18,12 @@ import (
 
 // URLBase is the root URL for the version of the docs associated with this
 // binary.
-var URLBase = "https://www.cockroachlabs.com/docs/" + build.BinaryVersionPrefix()
+var URLBase = "https://www.cockroachlabs.com/docs/" + build.VersionForURLs()
 
 // URLReleaseNotesBase is the root URL for the release notes for the .0 patch
 // release associated with this binary.
 var URLReleaseNotesBase = fmt.Sprintf("https://www.cockroachlabs.com/docs/releases/%s.0.html",
-	build.BinaryVersionPrefix())
+	build.VersionForURLs())
 
 // URL generates the URL to pageName in the version of the docs associated
 // with this binary.

--- a/pkg/server/server_test.go
+++ b/pkg/server/server_test.go
@@ -700,6 +700,8 @@ func TestServeIndexHTML(t *testing.T) {
 	unlinkFakeUI := func() {
 		ui.HaveUI = false
 	}
+	major, minor := build.BranchReleaseSeries()
+	version := fmt.Sprintf("v%d.%d", major, minor)
 
 	t.Run("Insecure mode", func(t *testing.T) {
 		srv := serverutils.StartServerOnly(t, base.TestServerArgs{
@@ -759,7 +761,7 @@ Binary built without web UI.
 			expected := fmt.Sprintf(
 				`{"Insecure":true,"LoggedInUser":null,"Tag":"%s","Version":"%s","NodeID":"%d","OIDCAutoLogin":false,"OIDCLoginEnabled":false,"OIDCButtonText":"","FeatureFlags":{"can_view_kv_metric_dashboards":true},"OIDCGenerateJWTAuthTokenEnabled":false,"LicenseType":"OSS","SecondsUntilLicenseExpiry":0,"IsManaged":false}`,
 				build.GetInfo().Tag,
-				build.BinaryVersionPrefix(),
+				version,
 				1,
 			)
 			require.Equal(t, expected, string(respBytes))
@@ -787,7 +789,7 @@ Binary built without web UI.
 				fmt.Sprintf(
 					`{"Insecure":false,"LoggedInUser":"authentic_user","Tag":"%s","Version":"%s","NodeID":"%d","OIDCAutoLogin":false,"OIDCLoginEnabled":false,"OIDCButtonText":"","FeatureFlags":{"can_view_kv_metric_dashboards":true},"OIDCGenerateJWTAuthTokenEnabled":false,"LicenseType":"OSS","SecondsUntilLicenseExpiry":0,"IsManaged":false}`,
 					build.GetInfo().Tag,
-					build.BinaryVersionPrefix(),
+					version,
 					1,
 				),
 			},
@@ -796,7 +798,7 @@ Binary built without web UI.
 				fmt.Sprintf(
 					`{"Insecure":false,"LoggedInUser":null,"Tag":"%s","Version":"%s","NodeID":"%d","OIDCAutoLogin":false,"OIDCLoginEnabled":false,"OIDCButtonText":"","FeatureFlags":{"can_view_kv_metric_dashboards":true},"OIDCGenerateJWTAuthTokenEnabled":false,"LicenseType":"OSS","SecondsUntilLicenseExpiry":0,"IsManaged":false}`,
 					build.GetInfo().Tag,
-					build.BinaryVersionPrefix(),
+					version,
 					1,
 				),
 			},

--- a/pkg/sql/logictest/testdata/logic_test/notice
+++ b/pkg/sql/logictest/testdata/logic_test/notice
@@ -63,5 +63,5 @@ UNLISTEN temp
 ----
 NOTICE: unimplemented: CRDB does not support LISTEN, making UNLISTEN a no-op
 HINT: You have attempted to use a feature that is not yet implemented.
-See: https://go.crdb.dev/issue-v/41522/v24.2
+See: https://go.crdb.dev/issue-v/41522/dev
 SQLSTATE: 0A000

--- a/pkg/sql/pgwire/testdata/pgtest/multiple_active_portals
+++ b/pkg/sql/pgwire/testdata/pgtest/multiple_active_portals
@@ -457,7 +457,7 @@ ReadyForQuery
 {"Type":"PortalSuspended"}
 {"Type":"DataRow","Values":[{"text":"2"}]}
 {"Type":"PortalSuspended"}
-{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/v24.2"}
+{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/dev"}
 {"Type":"ReadyForQuery","TxStatus":"E"}
 
 send crdb_only
@@ -504,7 +504,7 @@ ReadyForQuery
 {"Type":"BindComplete"}
 {"Type":"DataRow","Values":[{"text":"f"}]}
 {"Type":"PortalSuspended"}
-{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/v24.2"}
+{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/dev"}
 {"Type":"ReadyForQuery","TxStatus":"E"}
 
 send crdb_only
@@ -553,7 +553,7 @@ ReadyForQuery
 {"Type":"BindComplete"}
 {"Type":"DataRow","Values":[{"text":"10"}]}
 {"Type":"PortalSuspended"}
-{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/v24.2"}
+{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/dev"}
 {"Type":"ReadyForQuery","TxStatus":"E"}
 
 send crdb_only
@@ -602,7 +602,7 @@ ReadyForQuery
 {"Type":"BindComplete"}
 {"Type":"DataRow","Values":[{"text":"1"}]}
 {"Type":"PortalSuspended"}
-{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/v24.2"}
+{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/dev"}
 {"Type":"ReadyForQuery","TxStatus":"E"}
 
 send crdb_only
@@ -946,7 +946,7 @@ ReadyForQuery
 {"Type":"PortalSuspended"}
 {"Type":"DataRow","Values":[{"text":"10"}]}
 {"Type":"PortalSuspended"}
-{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/v24.2"}
+{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/dev"}
 {"Type":"ReadyForQuery","TxStatus":"E"}
 
 send

--- a/pkg/sql/pgwire/testdata/pgtest/portals_crbugs
+++ b/pkg/sql/pgwire/testdata/pgtest/portals_crbugs
@@ -367,6 +367,6 @@ ErrorResponse
 {"Type":"BindComplete"}
 {"Type":"DataRow","Values":[{"text":"(1,1)"}]}
 {"Type":"PortalSuspended"}
-{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/v24.2"}
+{"Type":"ErrorResponse","Code":"0A000","Message":"unimplemented: the statement for a pausable portal must be a read-only SELECT query with no sub-queries or post-queries","Detail":"cannot execute a portal while a different one is open","Hint":"You have attempted to use a feature that is not yet implemented.\nSee: https://go.crdb.dev/issue-v/98911/dev"}
 
 subtest end

--- a/pkg/ui/ui.go
+++ b/pkg/ui/ui.go
@@ -170,11 +170,12 @@ func Handler(cfg Config) http.Handler {
 		}
 		licenseTTL := base.LicenseTTL.Value()
 		oidcConf := cfg.OIDC.GetOIDCConf()
+		major, minor := build.BranchReleaseSeries()
 		args := indexHTMLArgs{
 			Insecure:         cfg.Insecure,
 			LoggedInUser:     cfg.GetUser(r.Context()),
 			Tag:              buildInfo.Tag,
-			Version:          build.BinaryVersionPrefix(),
+			Version:          fmt.Sprintf("v%d.%d", major, minor),
 			OIDCAutoLogin:    oidcConf.AutoLogin,
 			OIDCLoginEnabled: oidcConf.Enabled,
 			OIDCButtonText:   oidcConf.ButtonText,

